README.md

```diff
@@ -4,7 +4,7 @@
 
 The `opensource-website` project is the code and data behind [opensource.newrelic.com](https://opensource.newrelic.com) generated by [Gatsby](https://www.gatsbyjs.org).
 
-![CI](https://github.com/newrelic/opensource-website/workflows/CI/badge.svg) ![GitHub release (latest SemVer including pre-releases)](https://img.shields.io/github/v/release/newrelic/opensource-website?include_prereleases&sort=semver) [![Snyk](https://snyk.io/test/github/newrelic/opensource-website/badge.svg)](https://snyk.io/test/github/newrelic/opensource-website)
+![CI](https://github.com/newrelic/opensource-website/actions/workflows/pr.yml/badge.svg) ![GitHub release (latest SemVer including pre-releases)](https://img.shields.io/github/v/release/newrelic/opensource-website?include_prereleases&sort=semver) [![Snyk](https://snyk.io/test/github/newrelic/opensource-website/badge.svg)](https://snyk.io/test/github/newrelic/opensource-website)
```

Comment:
```suggestion
![CI](https://github.com/newrelic/opensource-website/actions/workflows/release.yml/badge.svg) ![GitHub release (latest SemVer including pre-releases)](https://img.shields.io/github/v/release/newrelic/opensource-website?include_prereleases&sort=semver) [![Snyk](https://snyk.io/test/github/newrelic/opensource-website/badge.svg)](https://snyk.io/test/github/newrelic/opensource-website)
```

Reply:
Let me double check with my team on which action this badge should be referencing, I'm thinking it may be the release.yml action.

Reply:
@rudouglas any thoughts good sir?

Reply:
I think it should actually be for the `Project Stats CI` workflow, but it would be good to have one for all 3 :D 

```suggestion
[![Project-Stats CI](https://github.com/newrelic/opensource-website/actions/workflows/project-stats.yml/badge.svg?event=schedule)](https://github.com/newrelic/opensource-website/actions/workflows/project-stats.yml) [![Pull Request](https://github.com/newrelic/opensource-website/actions/workflows/pr.yml/badge.svg)](https://github.com/newrelic/opensource-website/actions/workflows/pr.yml) [![Release](https://github.com/newrelic/opensource-website/actions/workflows/release.yml/badge.svg?event=push)](https://github.com/newrelic/opensource-website/actions/workflows/release.yml) ![GitHub release (latest SemVer including pre-releases)](https://img.shields.io/github/v/release/newrelic/opensource-website?include_prereleases&sort=semver) [![Snyk](https://snyk.io/test/github/newrelic/opensource-website/badge.svg)](https://snyk.io/test/github/newrelic/opensource-website)
```